### DISCUSSION TITLE
Branch Pushes to if-plugins/if

### DISCUSSION THREAD

**Discussion by maddendeavor** [Green-Software-Foundation/hack#131] [Ask Anything]
Hi, part of our project is updating the `if-plugin` repo and possibly the `if` repo as well (if we want to integrate params).  Currently, I can't push my branch up to if-plugins, can you add me to these repos or else let me know how you prefer us to upload our changes?

Thanks!
Christy

**Comment by jmcook1186** [comment]
Hi @maddendeavor - we would prefer you to keep your plugins in your own repositories rather than raising PRs against ours. This is because we want to encourage a wide ecosystem to plugins to grow organically without being limited by our capacity to maintain them.

**Reply by maddendeavor** [reply]
@jmcook1186 As per one of the suggestions of @jawache in this idea writeup: https://github.com/Green-Software-Foundation/hack/discussions/82 

We started working our project as an update to the cloud-metadata if-plugin. We picked this approach since this plugin seems to already hold a lot of datacenter info and because we are not super familiar with Node.js.  If we need to spin our own plugin let us know.  I guess another option could be forking off the if-plugin repo.

**Reply by jmcook1186** [reply]
yea my suggestion would be to fork the plugins repository and branch off it. You can provide the link to your branch in the hackathon submission. That way we get the diff and the commit history to evaluate but also we are not mixing hackathon projects with our IF core issue/pr boards.

Does that work for you?
